assets/src/components/media/manager/MediaManager.tsx

```diff
@@ -280,7 +280,13 @@ export class MediaManager extends React.PureComponent<MediaManagerProps, MediaMa
           })
           .then(() => this.setState({ error: Maybe.nothing<string>() }));
       })
-      .catch((e: Error) => this.setState({ error: Maybe.just(e.message) }))
+      .catch((e: Error | string) => {
```

Comment:
This error comes in as a string sometimes and was not always displaying previously.